### PR TITLE
Update SELinux policy for hirte

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

File: bluechi.spec.in [hunk 1]
```diff
@@ -127,19 +127,28 @@ SELinux policy associated with the bluechi and bluechi-agent daemons
 %{_mandir}/man8/bluechi*selinux.*
 
 %post selinux
+# Remove hirte policy
+if [ $1 -eq 1 ]; then
+semanage port -N -d -p udp 842 2>/dev/null || true
```
Comment (dougsland) [inline bluechi.spec.in]: looks like is missing a space in these semanage commands?

Comment (mwperina) [reply bluechi.spec.in]: Aren't those changes around hirte policies handled by `%postun` section in hirte RPMs? Aren't during upgrade hirte packages uninstalled at the beginning and only after that bluechi packages are installed?

Comment (rhatdan) [reply bluechi.spec.in]: @dougsland I don't see a missing space?

@mwperina bluchi.pp gets installed before hirte is uninstalled.

Comment (mwperina) [reply bluechi.spec.in]: Ah, OK. I missed that, thanks

**Review by mwperina — APPROVED** [review]
LGTM

**Comment by rhatdan** [comment]
@dougsland Have you successfully updated a running hirte system to a bluechi system, and seen no failures.

Currently our updates end up ignoreing hirte.conf and hirte-agent.conf files, so it is not seemless, but users need to modify the conf files.

**Comment by dougsland** [comment]
Agreed @rhatdan we need to add a logic to handle the config files and not only that, also put the services in the same state hirte was previously. For example, I had hirte/hirte-agent working and after this upgrade the new services are down... with incorrect config.

```
[root@control 08-16-2023]# dnf install *
Last metadata expiration check: 0:29:32 ago on Wed Aug 16 13:39:52 2023.
Dependencies resolved.
=====================================================================================================================================
 Package                            Architecture      Version                                          Repository               Size
=====================================================================================================================================
Installing:
 bluechi                            x86_64            0.5.0-0.202308161402.gitf21844e.el9              @commandline             63 k
     replacing  hirte.x86_64 0.4.0-1.el9
 bluechi-agent                      x86_64            0.5.0-0.202308161402.gitf21844e.el9              @commandline             62 k
     replacing  hirte-agent.x86_64 0.5.0-0.202308101459.gitb3d0dec.el9
 bluechi-agent-debuginfo            x86_64            0.5.0-0.202308161402.gitf21844e.el9              @commandline            115 k
 bluechi-ctl                        x86_64            0.5.0-0.202308161402.gitf21844e.el9              @commandline             33 k
     replacing  hirte-ctl.x86_64 0.4.0-1.el9
 bluechi-ctl-debuginfo              x86_64            0.5.0-0.202308161402.gitf21844e.el9              @commandline             61 k
 bluechi-debuginfo                  x86_64            0.5.0-0.202308161402.gitf21844e.el9              @commandline            135 k
 bluechi-debugsource                x86_64            0.5.0-0.202308161402.gitf21844e.el9              @commandline             76 k
 bluechi-selinux                    x86_64            0.5.0-0.202308161402.gitf21844e.el9              @commandline             25 k
     replacing  hirte-selinux.x86_64 0.5.0-0.202308101459.gitb3d0dec.el9
 python3-bluechi                    x86_64            0.5.0-0.202308161402.gitf21844e.el9              @commandline             22 k
Installing dependencies:
 python3-dasbus                     noarch            1.4-5.el9                                        appstream               102 k

Transaction Summary
=====================================================================================================================================
Install  10 Packages

Total size: 695 k
Total download size: 102 k
Is this ok [y/N]: y


<SNIP>
# systemctl status bluechi
○ bluechi.service - BlueChi systemd service controller manager daemon
     Loaded: loaded (/usr/lib/systemd/system/bluechi.service; disabled; preset: disabled)
     Active: inactive (dead)
       Docs: man:bluechi(1)
             man:bluechi.conf(5)


```

**Comment by mwperina** [comment]
I'd not over-engineered this, I don't think anyone uses the project in production yet, so the smooth package upgrade should be enough, I'd not invest the time to (for example) convert hirte config files to bluechi. I believe we have more important work around integration tests ...

**Comment by dougsland** [comment]
> I'd not over-engineered this, I don't think anyone uses the project in production yet, so the smooth package
> upgrade should be enough, I'd not invest the time to (for example) convert hirte config files to bluechi. I believe
> we have more important work around integration tests ...

We need to track it, opened an issue for it.

I am ready to merge Dan's patch but I would like to confirm if I am the only one that see the commands after the `if statement` with no indentation.  (pointed in this commit/patch)

**Comment by rhatdan** [comment]
Added the indent,  I thought you were complaining about the spaces in the actual `if [ ...` line

**Comment by dougsland** [comment]
> Added the indent, I thought you were complaining about the spaces in the actual `if [ ...` line

Sorry @rhatdan , bad English on my side. :/

**Comment by dougsland** [comment]
LGTM

**Pull request merged**